### ISSUE TITLE
Evaluate the results of a git commit

### ISSUE THREAD

**Issue by dennischristmann** [oxygenxml/oxygen-git-plugin#41]
In our workflow, we use git hooks to check the validity of commit messages (e.g., they must contain a valid issue ID). The git integration also seems to run the hooks, so a commit is rejected, if the commit message is not valid. 
Though the commit is not applied, the git staging view tells erroneously that everything was fine. Like the git integration in Egit/Eclipse, it would be better to evaluate the return values of the git hooks and to display the error messages if the commit went wrong.

**Comment by kirkilj** [comment]
There may be another reason to check for the results of a commit. I haven't been able to reproduce it yet, but since the latest update I've had 2 instances where I received the commit successful message, but the changes listed in the stage did not clear out. Doing a git status or git log from the CL verified that the commit did not occur. In the most recent instance, yesterday, I had staged 50 or so image file deletions. When I tried to recreate it, the problem didn't occur. I'll keep an eye out and record specific details if it happens again.
